ipyleaflet/leaflet.py

```diff
@@ -611,13 +631,18 @@ def _get_data(self):
             self.value_max = max(self.choro_data.items(), key=lambda x: x[1])[1]
 
         colormap = self.colormap.scale(self.value_min, self.value_max)
-        color_dict = {key: colormap(self.choro_data[key]) for key in self.choro_data.keys()}
-
         data = copy.deepcopy(self.geo_data)
-        for feature in data['features']:
-            feature['properties']['style'] = dict(fillColor=color_dict[feature['id']],
-                                                  color=self.border_color,
-                                                  weight=0.9)
+
+        if self.style_callback:
```

Comment:
I wonder if it would not be cleaner to use a default value for `style_callback` in the case of the `Choropleth` class, something like:

```python
@default('style_callback')
def _default_style_callback(self):
    def compute_style(feature, colormap, choro_data):
        return dict(
            fillColor=colormap(choro_data),
            color='black',
            weight=0.9
        )
    return compute_style
```

And we could remove this `border_color` attribute if needed.

Reply:
Added a default value for style_callback. Thanks